backend/src/zimfarm_backend/db/language.py

```diff
@@ -32,16 +32,22 @@ def get_languages(
 
     return LanguageListResult(
         nb_languages=count_from_stmt(session, query),
-        languages=[
-            LanguageSchema(
-                code=language_code,
-                name_en=language_name_en,
-                name_native=language_name_native,
-            )
-            for (
-                language_code,
-                language_name_en,
-                language_name_native,
-            ) in session.execute(query.offset(skip).limit(limit)).all()
-        ],
+        languages=sorted(
+            [
+                LanguageSchema.model_validate(
+                    {
+                        "code": language_code,
+                        "name_en": language_name_en,
+                        "name_native": language_name_native,
+                    },
+                    context={"skip_validation": True},
```

Comment:
What's the rationale for validating but skipping the validation? Is it that it's assumed the DB data is valid? What's difference with previous version?

Reply:
The DB data can sometimes be "invalid" owing to constraints changing. So, while reading from DB, we skip validating. But while updating or adding, we don't. Some of the data in the db fails the validation constraints and it won't be feasible to start modifying them. For example, the language code should be between length 2 and 8 but it's been observed some existing data is of length 12 and pydantic fails. So, we use context to skip validation on reads.